covalent/elrond-node-obs

```diff
@@ -0,0 +1,29 @@
+FROM golang:1.15.7 as builder
+
+RUN git clone https://github.com/ElrondNetwork/elrond-config-mainnet && cd elrond-config-mainnet && git checkout --force tags/v1.2.22.1
+RUN git clone https://github.com/ElrondNetwork/elrond-go.git && cd elrond-go && git checkout --force tags/v1.2.22
```

Comment:
we should not bind to specific versions. What happens if we do a release and forget to update this?

Reply:
refactored.